client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -22,6 +22,10 @@ import { deleteTerm } from 'calypso/state/terms/actions';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 
 class TaxonomyManagerListItem extends Component {
+	constructor( props ) {
+		super( props );
+		this.countRef = createRef();
+	}
```

Comment:
We usually write this as an instance field initialization:
```js
class Taxonomy extends Component {
  countRef = createRef();

  render() {
    ...
  }
}
```

Reply:
Cool, that's much better than the constructor :+1: